build.sbt

```diff
@@ -97,10 +97,14 @@ lazy val commonSettings = Seq(
   crossPaths := false,
   publishTo :=  Some("releases" at "https://s01.oss.sonatype.org/" + "service/local/staging/deploy/maven2"),
   publishMavenStyle := true,
-  version := "1.0.1",
+  version := "1.1.0",
   scalaVersion := "3.2.2",
   sbtVersion := "1.8.2",
-  scalacOptions := Seq("-unchecked", "-deprecation", "-feature"),
+  scalafixOnCompile := true,
```

Comment:
I think there is a gotcha in the tool documentation, that a linter pass should be done before testing in CI or the linter might impact the results of said tests.